### PR TITLE
New ci pipeline

### PR DESCRIPTION
**Pull request by mulf0**

### CODE REVIEW

**Comment by mulf0** [comment]
new environment vars for prod:

- [x] Vercel: VERCEL_WEBHOOK_URL
- [x] Digital: Ocean: CMS_DEPLOY_KEY

**Comment by kempsterrrr** [comment]
>  Vercel: VERCEL_WEBHOOK_URL
Digital: Ocean: CMS_DEPLOY_KEY 

do you have the permission to do these mate or do you need me to do?

File: .github/workflows/build-deploy.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+on:
+  push:
+    branches: ["main"]
+env:
+  DAY_OF_WEEK: Monday
+  APP_KEYS:value_1,value_2,value_3
+  API_TOKEN_SALT:API_TOKEN_SALT_VALUE
+  ADMIN_JWT_SECRET:ADMIN_JWT_SECRET_VALUE
+  JWT_SECRET:__SKIP_FOR_LOCAL_ENVIRONMENT--GENERATED__
+jobs:
```
Comment (ntindle) [inline .github/workflows/build-deploy.yml:12]: These should be accessible and not need renamed

File: packages/cms/scripts/deploy.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  try {
+    const { data } = await axios.post(
+      'https://api.digitalocean.com/v2/apps/21030157-a72b-40d2-afa6-0b498c63144d/deployments',
+      {
+        force_build: true,
+      },
+      {
+        headers: {
+          'Content-Type': 'application/json',
+          Authorization: `Bearer ${process.env.CMS_DEPLOY_KEY}`,
```
Comment (ntindle) [inline packages/cms/scripts/deploy.js:14]: Careful with this. Would plan to get this from something else long term like a vault or secure key value store

**Review by ntindle — APPROVED** [review]

File: packages/cms/scripts/deploy.js (hunk 2 above)
Comment (mulf0) [reply packages/cms/scripts/deploy.js:14]: I have heard about these in passing. Setting up the github action secrets mentioned using libsodium. so something along those lines? Good callout for the future and would love some help when we get to that point.

File: .github/workflows/build-deploy.yml (hunk 1 above)
Comment (mulf0) [reply .github/workflows/build-deploy.yml:12]: not sure what you mean here?

Comment (ntindle) [reply .github/workflows/build-deploy.yml:12]: No point in doing the mapping here is what I'm saying, you can just use the values directly

**Pull request merged**